manifests/0000_60_cluster-platform-operator-manager_00-platformoperator.crd.yaml

```diff
@@ -12,82 +13,82 @@ spec:
     listKind: PlatformOperatorList
     plural: platformoperators
     shortNames:
-    - platform
-    - platforms
+      - platform
+      - platforms
```

Comment:
Seems like YQ had some opinions on how to format an array, I'll check if I can override this setting.

Reply:
[Related YQ issue](https://github.com/mikefarah/yq/issues/194)

Reply:
Fixed by using [Kustomize commonAnnotations](https://github.com/kubernetes-sigs/kustomize/blob/672c751715be7dd0b43b4a2fce956c84452e0db9/api/types/kustomization.go#L59-L60)